src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterService.java

```diff
@@ -178,6 +178,21 @@ private PGPPublicKey loadPgpPublicKey(String encryptionPublicKey) {
         }
     }
 
+    private int getCopies(ILetterRequest letter) {
+        if (letter instanceof LetterRequest) {
+            return ((LetterRequest) letter).documents.size();
+        } else if (letter instanceof LetterWithPdfsRequest) {
+            return ((LetterWithPdfsRequest) letter).documents.size();
+        } else if (letter instanceof LetterWithPdfsAndNumberOfCopiesRequest) {
+            return copies.apply((LetterWithPdfsAndNumberOfCopiesRequest) letter);
+        } else {
+            throw new UnsupportedLetterRequestTypeException();
```

Comment:
This doesnt have code coverage

Reply:
as exception will not come, will break in previous call itself. So removed this line.